src/Core/Models/Request/SetKeyConnectorKeyRequest.cs

```diff
@@ -7,7 +7,7 @@ public class SetKeyConnectorKeyRequest
     {
         public string Key { get; set; }
         public KeysRequest Keys { get; set; }
-        public KdfType Kdf { get; set; }
+        public KdfType? Kdf { get; set; }
```

Comment:
This shouldn't be null, [server side](https://github.com/bitwarden/server/blob/cb1ba50ce26ce33b2a5acf30536a2075e4fadebd/src/Api/Models/Request/Accounts/SetKeyConnectorKeyRequestModel.cs#L16) expects always a value and it's required.
I've checked and you can return to the `GetValueOrDefault(...)` on line 20 because previously the plain parameter of the type was not null. Sorry about the confusion

Reply:
Done in 88c554c91701ddc33ced7743ec029ef3fee9f8f0.